career-planning-tool/index.html

```diff
@@ -88,65 +88,89 @@ <h3>GS Level</h3>
 
-                        {% for compet in competency_group_array %}
-                            {% if sorted_competency_group_array contains compet %}
-
-                            {% else %}
-                                {% assign sorted_competency_group_array = sorted_competency_group_array | push: compet %}
-                            {% endif %}
+                            {% assign competency_group_id = competency_group | replace: ", ", "-" | replace: " ", "-" %}
+                            <div class="career-competency-and-competency-group-input-groups margin-bottom-2">
+                                <div class="career-competency-level-2-input-group padding-1 text-bold text-black bg-white radius-lg border-2px border-gray-30 flex-column">
+                                    <div class="career-competency-toggle-open">
+                                        <div class="career-competency-toggle-open--inner">
+                                        <button type="button" id="competency-group-button-{{ competency_group_id | downcase }}"  class="usa-button usa-button--unstyled text-no-underline text-black career-competency-toggle-button" title="{{ competency_group }} Open"><i class='fa fa-plus'></i> <b>{{ competency_group }}</b></button>
+                                        <div class="career-competency-level-3-input-group">
+                                            <div class="career-input-group">
+                                              <label for="{{ competency_group_id | downcase }}" class="career-competency-input-groups-label text-underline text-accent-cool-darker display-inline-block font-sans-3xs"><strong class="cpt-strong">Select All</strong></label>
+                                                <input
+                                                    type="checkbox"
+                                                    id="{{ competency_group_id | downcase}}"
+                                                    data-filter="competency"
+                                                    data-major-group="job-specific"
+                                                    data-id="{{ competency_group_id | downcase }}"
+                                                    title="{{ competency_group }}"
+                                                    style="position: absolute; left: -9999px;"
+                                                />
+                                            </div>
+                                        </div>
+                                        </div>
+                                        {% for competency in competency_array %}
+                                        {% assign competency_id = competency | replace: ", ", "-" | replace: " ", "-" %}
+                                        <div class="career-competency-input-groups" style="display: none;">
```

Comment:
please us custom classes or USWDS utilities classes

Reply:
This style is temporary. It will be overridden based on user choices.